### ISSUE TITLE
Deck searching not working.

### ISSUE THREAD

**Issue by jjulio93** [magefree/mage#4314]
When any effect that would cause you to search your library gets stuck after passing priority back to you. Often the message will show that you are waiting for an opponent to pass priority to you meanwhile the other players are waiting for you to search your library.

This was tested only in commander free-for-all matches.

**Comment by LevelX2** [comment]
Is this a new problem since updating client to 1.4.26V10 or did it happen also with client V9?

What are example cards with search effects that caused it?

Can you explain more in detail what "gets stuck after passing priority back to you" exactly means?
The problem occurs after you selected a card from your library or before?

**Comment by jjulio93** [comment]
The problem began since updating the client to 1.4.26v10, it has never happened before.

I tested with Windswept Heat, and Kodama´s Reach. 

The game indicates that i´m waiting for other player to pass priority back to me, meanwhile the other players report that they are waiting for me to search my library, but the window never opens.

Other detail seems to be that the log indicates that i began searching my library.

Tested this problem with some friends, and they have the same problem to.
